### DISCUSSION TITLE
In Nextjs when I use grapesjs got TypeError: Object.defineProperty called on non-object

### DISCUSSION THREAD

**Discussion by ibryanwu** [GrapesJS/grapesjs#5961] [Q&A]
Hello
The thing is that when I try to initialize grapesjs, I get the following error: 
TypeError: Object.defineProperty called on non-object 

```
import grapesjs from 'grapesjs';
...
useEffect(() => {
    const editor = grapesjs.init({
      container: '#gjs',
      // other options
    });
  }, []);

```

However,  I use [grapesjs="https://unpkg.com/grapesjs"] It works.

Someone can help?
THANKS

**Comment by artf** [comment]
Please provide a reproducible demo (eg. on Stackblitz)
